src/obfl-specification.html

```diff
@@ -2759,6 +2759,74 @@ <h4 id="L21851">Example</h4>
   </tbody>
 </table>
   
+<h3 id="L2762">mark-capital-letters</h3>
+
+<p>The overall policy for marking capital letters should be determined by the
+implementing application, for example per job or in a system setting. A user's
+preference should be respected, unless doing so would render the text difficult
+to read or interpret for some reason. The hyphenate attribute can be used when
+detailed control over hyphenation is required, e.g. to ensure proper rendering of
+content that may be difficult to interpret if capital markings are used.</p>
+
+<p>The marking of capital letters specified by <code>mark-capital-letters</code>
+applies to all elements in its content unless overridden with another instance of
+<code>mark-capital-letters</code>. In particular, the empty value of
+<code>mark-capital-letters</code> is used on an element B to override a
+specification of <code>mark-capital-letters</code> on an enclosing element A,
+without specifying another marking policy. Within B, the user's
+preference for marking capital letters should be applied.</p>
+
+<p>The value of the <code>mark-capital-letters</code> attribute can either be 'true' or 'false'.
+The value 'true' indicates that a text algorithm add capital markings to the
+text contents. The value 'false' indicates that a text algorithm should
+<strong>NOT</strong> add capital markings text contents.</p>
+
+<h4 id="L2784">Example</h4>
+
+<p>The following table illustrates the output for different combinations of
+input text and capital letters marking attribute value:</p>
```

Comment:
I don't know about this table. The attribute just enables or disables indication of capitals, there's not much to understand about that. So to me, this example basically just boils down to an example of how capitals could be indicated in braille. Especially rows 3 to 6 are beside the point IMO. They just go into more detail about how this particular braille system chosen in the example works, which is irrelevant for OBFL.